### ISSUE TITLE
Browser Testing

### ISSUE THREAD

**Issue by elbotho** [serlo/frontend#81]
We should do some browser testing, 
I think we have Chrome and Firefox covered, which leaves all time favourites like
IE, Safari and div. mobile browser.

### Quick checks desktop:
- 🟢current **Chrome/Firefox** on mac
- 🟢relatively current **Safari** on mac (after #83 is done)
- 🟢current **Edge** on Mac: Thankfully it's basically Chrome.
- 🟠 **IE11** on Win: Inconsistencies but usable
- 🟢**Firefox/Chrome/Brave** on Linux/Windows: Do we have those covered? (@Entkenntnis, @kulla)
- 🟢**Opera** on Mac: Looks good!

### Quick checks mobile:
- 🟢Mobile Preview in Chrome and Firefox
- 🟠Firefox and stock Chrome on Android: Weird colors (will investigate), bit sluggish, [sec-menu ux](#59 ) really bad but looks fine.
- 🟢Safari, iPhone? (looks good)
- ⚪iPad: (postponed)

### Current Stats:
![image](https://user-images.githubusercontent.com/1258870/80947398-65ee3980-8df0-11ea-82ec-2602bda95906.png)
so imo for opera + ie11 it would be nice to have it at least working.

**Comment by kulla** [comment]
Which kind of tests do you want to perform? That the site loads in each browser or that each component renders the same in each browser?

**Comment by elbotho** [comment]
Dreaded browser inconsistencies and bugs.

**Comment by Entkenntnis** [comment]
I will take a look on IE 11

**Comment by kulla** [comment]
> Dreaded browser inconsistencies and bugs.

Okay, I see. I thought you meant e2e tests :-)

**Comment by Entkenntnis** [comment]
here are some screenshots

ex1:

![grafik](https://user-images.githubusercontent.com/13507950/80958603-98566180-8e05-11ea-94bd-db8dce41191c.png)

ex2:

![grafik](https://user-images.githubusercontent.com/13507950/80958716-b754f380-8e05-11ea-91b9-5948bba516b6.png)

ex3:

![grafik](https://user-images.githubusercontent.com/13507950/80958928-19adf400-8e06-11ea-9725-867dfcf45eb3.png)

Main problems are images in articles and taxonomy.

Javascript - functionality is quite ok, bit slow, but everything seems to work:

![grafik](https://user-images.githubusercontent.com/13507950/80958893-0e5ac880-8e06-11ea-8516-7a60a667d52c.png)

**Comment by elbotho** [comment]
@Entkenntnis Thanks for the screenshot, set up an VM and I think by now I got the worst of it. Good enough for me. Maybe you can check again:
- https://github.com/serlo/frontend/pull/155
- https://frontend-git-browser-fixes.serlo.now.sh/
